### PR TITLE
Revert "changed versioning 0.1.0 [major version].[minor version.[bug fixes]"

### PR DESCRIPTION
**Pull request by stannlee**
Reverts Opensource-254/Opensource-254.github.io#9
Hi, @azim254 JQuery is not required we are using [Bootstrap vue](https://bootstrap-vue.js.org/)

### CODE REVIEW

**Comment by stannlee** [comment]
@azim254 please make those changes in the  revert-9-master branch

File: package.json [hunk 1]
```diff
@@ -16,11 +16,9 @@
     "@nuxtjs/axios": "^5.3.6",
     "@nuxtjs/dotenv": "^1.4.0",
     "@nuxtjs/pwa": "^3.0.0-0",
-    "bootstrap": "^4.4.1",
+    "bootstrap": "^4.1.3",
```
Comment (BILLthebuilder) [inline package.json]: Is there a  reason this has been downgraded?

Comment (azim254) [reply package.json]: nme run na latest ina brick

Comment (stannlee) [reply package.json]: I'll upgrade that in a few though it has no effect

**Comment by stannlee** [comment]
@azim254 Github actions inatumia jest and it works poa. Raise an issue tuangalie

**Pull request merged**